wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -958,6 +980,19 @@ private void broadcastBlockchainState() {
         }
     }
 
+    private int percentageSync() {
+        int chainHeadHeight = blockChain.getChainHead().getHeight();
+        int mostCommonChainHeight;
+        if (peerGroup.getMostCommonChainHeight() > 0) {
```

Comment:
This will generate a NullPointerException if peerGroup is null.

Reply:
How to make it null? I've tried to turn off my internet connection and clear blockchain data but it's still not null

Reply:
In my case, I found it to be **null** when the testnet app was taking a long time to connect and when I turned off the network/mobile/data on the emulator.